integration_tests/macros/e2e_tests/test_config_levels.sql

```diff
@@ -0,0 +1,20 @@
+{% test config_levels(model, expected_config, time_bucket) %}
+    {%- if execute and flags.WHICH in ['test', 'build'] %}
+        {%- set unexpected_config = [] %}
+        {%- set model_relation = dbt.load_relation(model) %}
+        {% set model_graph_node = elementary.get_model_graph_node(model_relation) %}
+
+        {%- set time_bucket = elementary.get_time_bucket(time_bucket, model_graph_node) %}
+        {%- if time_bucket != expected_config.get('time_bucket') %}
+            {%- set unexpected_message = ('got config: {0}, expected config: {1}').format(time_bucket, expected_config.get('time_bucket') ) %}
```

Comment:
Why is `unexpected_config` instead of raising an error here?

Reply:
Helps to debug, you can see clearly what was unexpected on the error message (if more than one config you will see all).